app/controllers/api/vms_controller.rb

```diff
@@ -8,6 +8,9 @@ class VmsController < BaseController
     include Subcollections::Software
     include Subcollections::Snapshots
 
+    VALID_EDIT_ATTRS = %w(description child_relationships parent_relationship).freeze
```

Comment:
would prefer child_resources and parent_resource for consistency with other _resource   exposure in API calls.